tests/acceptance/extract-i18n.spec.js

```diff
@@ -0,0 +1,51 @@
+const tmp = require('../helpers/tmp');
+const ng = require('../helpers/ng');
+const existsSync = require('exists-sync');
+const path = require('path');
+const root = process.cwd();
+const expect = require('chai').expect;
+
+// TODO: Enable when tests get validated
+describe.skip('Acceptance: ng xi18n', function() {
+  beforeEach(function() {
+    this.timeout(180000);
+    return tmp.setup('./tmp').then(function() {
+      process.chdir('./tmp');
+    }).then(function() {
+      return ng(['new', 'foo', '--link-cli']);
```

Comment:
`ng new` should be executed with `--skip-npm` and `--skip-bower` flags. But if we do so, the extractor won't work. I guess it is because the extractor seems to compile the project. If the dependencies are not installed the command throws an exception because eventually a module will not be found.

If the `beforeEach` has to install the dependencies it takes too much time. That's why I skipped the test because I though installing all vendors is unacceptable.

Any idea how to make this work?